core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar-namespace-elements/namespace-filter/namespace-filter.component.scss

```diff
@@ -26,10 +26,13 @@
     color: var(--ame-font);
     cursor: pointer;
     user-select: none;
+    opacity: 60%;
     border: solid 1px transparent;
```

Comment:
Add `box-sizing: border-box` rule so the size will change inside shape, not to outside